scripts/node/README.md

```diff
@@ -0,0 +1,42 @@
+# Steps to benchmark node scanning 
+
+## Pre steps:
+
+quay.io/repository/rhacs-eng/sandbox:benchmark-node-analyze-1.0 is an Ubi8 based image with vim, RPM and RPM-build installed. Those packages are necessary for benchmark node analyze function
+
+back-up image: `quay.io/rh_ee_yli3/nodes:benchmark-node-analyze-1.0`
+
+execute
+```
+docker run -it --rm quay.io/repository/rhacs-eng/sandbox:benchmark-node-analyze-1.0
+```
+---------------------------------------------------------------------------------------------
+
+(optional) create a temporary/target folder in the running ubi8 bash for RPM specs. e.g mkdir temp-specs
+
+1.go to the target folder and create a bash script to generate RPM specs (see scanner/scripts/node/generate-rpm-specs.sh)
+```
+vim generate-junit-reports.sh
+```
+2.execute the bash file: bash generate-rpm-specs.sh <number of the RPM specs>  . You will be able to see the RPM spec files in the folder
+```
+bash generate-rpm-specs.sh 100
+```
+3.execute
+```
+rpmbuild -bb *.spec
+```
+You will see output like: `Wrote: /root/rpmbuild/RPMS/x86_64/package_name9-0-0.x86_64.rpm` This gives you the path where RPM files were generated to.
+
+4.install all packages using the rpm files just created: yum localinstall <path to rpm files>/.*rpm  
+```
+yum localinstall /root/rpmbuild/RPMS/x86_64/.*rpm
+```
+5.check the running ubi containers by executing: `docker container ls` and get the container id of the container where you are currently installing the rpm packages
+
+6.execute `docker export [container id]>[tar name].tar` (e.g `docker export c8c57bb7e926>demoV1.tar`) and unzip the tar ball to get the file system
+
+7.Go to **scanner/blob/master/benchmarks/analyzeNode** folder and make sure to add local path to file system in line 13, then execute command:
```

Comment:
I believe you need [a space between `.` and the beginning of the list content](https://github.github.com/gfm/#lists).

Also, for all list items, I think the list contents are English sentences, and AFAIK, it would be better style to Capitalize the first letter.